Cargo.lock

```diff
@@ -798,7 +798,7 @@ dependencies = [
 [[package]]
 name = "osmesa-src"
 version = "17.4.0-devel"
-source = "git+https://github.com/servo/osmesa-src#1cf8c2f21c7e9047c3bd5a2f1accbc08e5636cf7"
+source = "git+https://github.com/jrmuizel/osmesa-src?branch=serialize#0f81d082b4cca16ea4d344bf6758b5a2f8d98277"
```

Comment:
why are we changing the osmesa path here?

Reply:
It's from https://github.com/servo/webrender/pull/2639 - some CI weirdness that @jrmuizel needed temporarily.